CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ option(EMBEDDINGS   "Embed developers assets"                     ON )
 option(PROFILING    "Enable internal profiling instruments"       OFF)
 option(BACKWARD     "Enable stacktrace logging instruments"       ON )
 option(CLEAR_OBJS   "Clear object files"                          OFF)
+option(EXTERNAL_PROJECT "Build external project"                  OFF)
```

Comment:
I believe the option has to be enabled for local environments by default.
We could do so via checking presence and value of env variables `$CI or $GITHUB_ACTIONS` and turning the option off for the case (to pass CI disk space limitations only).